kata.py

```diff
@@ -0,0 +1,69 @@
+#Imports
+import json
+
+class Parcel:
+    def __init__(self):
+        self.name=""
+        self.dimension=0
+        self.size=""
+        self.cost=0
+        self.weight=0
+
+    def create_parcel(self, myName, myDimension, myWeight):
+        self.name=myName
+        self.dimension=myDimension
+        self.weight=myWeight
+        self.get_size()
+        self.cost=self.sizeTocost(self.size)
+        self.addOverweight()
+
+    def get_size(self):
+        if(self.weight>50):
+            self.size="Heavy"
+        elif(self.dimension<10): 
+            self.size="s"
+        elif(self.dimension>10 and self.dimension<50): 
+            self.size="m"
+        elif(self.dimension>50 and self.dimension<100):
+            self.size="l"
+        elif(self.dimension>=100):
+            self.size="xl"
+        else:
+            raise ValueError("Dimensions cannot be negative")
```

Comment:
The if-else structure means that adding or changing parcel types can become problematic (needs updating in numerous places)

Reply:
@egnwd I wasn't sure how to do this differently. 
I think in C# I would have used switch cases, but wasn't sure what was the smart alternative here.

Reply:
You could encapsulate each parcel as an object/class and let each new parcel decide its logic for if it fits perhaps? - but my python isn't great, so I don't know the most idiomatic way - but I'm sure there's something